test/integration/spec/remotetracks.js

```diff
@@ -22,11 +22,16 @@ function getTracksOfKind(participant, kind) {
   return [...participant.tracks.values()].filter(remoteTrack => remoteTrack.kind !== kind).map(({ track }) => track);
 }
 
-describe('RemoteParticipant', function() {
+
+describe('RemoteVideoTrack', function() {
   // eslint-disable-next-line no-invalid-this
   this.timeout(60000);
 
-  describe('#setPriority', () => {
+  // eslint-disable-next-line no-warning-comments
+  // TODO: enable these tests when track_priority MSP is available in prod
+  (defaults.topology === 'peer-to-peer' || defaults.environment === 'prod'
```

Comment:
@makarandp0 On second thought, I think the following tests should run for all values of topology and environment:
```
- can setpriority null on audio track
- can setpriority high on audio track
- can setpriority low on audio track
- can setpriority standard on audio track
- can setpriority null on video track
- can setpriority high on video track
- can setpriority low on video track
- can setpriority standard on video track
- can setpriority null on data track
- can setpriority high on data track
- can setpriority low on data track
- can setpriority standard on data track
- throws for an invalid priority value: undefined
- throws for an invalid priority value: foo
- throws for an invalid priority value: 42
```
The other tests should be conditionally enabled.